apps/web/src/components/AccessRisk/SwapRevampRiskDisplay.tsx

```diff
@@ -0,0 +1,377 @@
+          {t('Reset Slippage')}
+        </StyledTextButton>
+        {/* <StyledLinkExternal color="primary60" external href="https://www.hashdit.io" showExternalIcon={false}>
+          {t('Learn More')}
+        </StyledLinkExternal> */}
+      </FlexGap>
+    </FlexGap>
+  )
+}
+
+export const RiskDetails: React.FC<RiskDetailsProps> = ({ token, riskLevelDescription }) => {
+  const { t } = useTranslation()
+  const { isDataLoading, riskLevel } = useRiskCheckData(token)
+  const { chainId } = useActiveChainId()
+
+  if (!isDataLoading && riskLevel && riskLevel <= TOKEN_RISK.SIGNIFICANT && riskLevel >= TOKEN_RISK.MEDIUM) {
+    if (riskLevel && riskLevel >= TOKEN_RISK.VERY_LOW && token?.address) {
+      return (
+        <FlexGap alignItems="flex-start" gap="8px">
+          <FlexGap justifyContent="center" alignItems="flex-start" flexDirection="column" gap="8px">
+            <Text>
+              {riskLevelDescription ||
+                t(
+                  'The address contains risk factors that are possible to lead to partial loss of funds, or medium to lower chance of catastrophic losses.',
+                )}
+            </Text>
+            <StyledLinkExternal href="https://www.hashdit.io">{t('Result provided by HashDit')}</StyledLinkExternal>
+            {chainId === ChainId.BSC && (
+              <StyledLinkExternal external href={`https://dappbay.bnbchain.org/risk-scanner/${token.address}`}>
+                {t('Get more details from RedAlarm')}
+              </StyledLinkExternal>
+            )}
+          </FlexGap>
+        </FlexGap>
+      )
+    }
+    return null
+  }
+  return null
+}
+
+export const useShouldRiskPanelDisplay = (token0?: ERC20Token, token1?: ERC20Token) => {
+  const { isDataLoading: isDataLoading0, riskLevel: riskLevel0 } = useRiskCheckData(token0)
+  const { isDataLoading: isDataLoading1, riskLevel: riskLevel1 } = useRiskCheckData(token1)
+  if ((isDataLoading0 && token0) || (isDataLoading1 && token1)) {
+    return false
+  }
+  return (
+    (riskLevel0 && riskLevel0 <= TOKEN_RISK.SIGNIFICANT && riskLevel0 >= TOKEN_RISK.MEDIUM) ||
+    (riskLevel1 && riskLevel1 <= TOKEN_RISK.SIGNIFICANT && riskLevel1 >= TOKEN_RISK.MEDIUM)
+  )
+}
+
+export const RiskDetailsPanel: React.FC<RiskDetailsPanelProps> = ({
+  token0,
+  token1,
+  token0RiskLevelDescription,
+  token1RiskLevelDescription,
+  isPriceImpactTooHigh,
+  isSlippageTooHigh,
+}) => {
+  const [isOpen, setIsOpen] = useState(false)
+  const [modalOpen, setModalOpen] = useState(false)
+  const isRiskToken0 = useShouldRiskPanelDisplay(token0)
+  const isRiskToken1 = useShouldRiskPanelDisplay(token1)
+  const isRiskMoreThanOne = useMemo(() => {
+    let count = 0
+    if (isRiskToken0) {
+      count++
+    }
+    if (isRiskToken1) {
+      count++
+    }
+    if (isPriceImpactTooHigh) {
+      count++
+    }
+    if (isSlippageTooHigh) {
+      count++
+    }
```

Comment:
Can we change it 

if (isRiskToken0 || isRiskToken1 || isPriceImpactTooHigh || isSlippageTooHigh) {
   count++
}